services/galley/test/integration/API.hs

```diff
@@ -3330,7 +3343,7 @@ deleteRemoteMemberConvLocalQualifiedOk = do
               *> mockReply [mkProfile qEve (Name "Eve")]
           ]
   (convId, _) <-
-    withTempMockFederator' (mockedResponse <|> mockReply ()) $
+    withTempMockFederator' ("get-not-fully-connected-backends" ~> NonConnectedBackends mempty <|> mockedResponse <|> mockReply ()) $
```

Comment:
general thought: should we create a few larger mock rule sets and give them names, so we can write many tests with them.  passing individual rules to each test feels like work.  not sure, though, maybe let's just see how this evolves.